docs/snippets/MULTI0.cpp

```diff
@@ -3,17 +3,23 @@
 int main() {
 //! [part0]
 ov::Core core;
+
+// Read a network in IR, PaddlePaddle, or ONNX format:
 std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
-// the "MULTI" device is (globally) pre-configured with the explicit option
-core.set_property("MULTI", ov::device::priorities("HDDL,GPU"));
```

Comment:
if delete these 2 lines, below does not work
ov::CompiledModel compileModel0 = core.compile_model(model, "MULTI");
1) we can compile with MULTI + device priority config
2) we can pre-set the config by core.set_property("MULTI", ov::device::priorities(*****));
    then, we can compile Model to MULTI as below:
    ov::CompiledModel compileModel0 = core.compile_model(model, "MULTI");

Reply:
Done. Additional comments changes should make things even clearer now. Thanks!